lib/drivers/docs/tandem.md

```diff
@@ -73,46 +76,50 @@ Device-specific? (Add any device-specific notes/additions here.)
-  - `*[-]` reservoir change (or reservoir rewind)
-    - `*[-]` needed to infer a suspend (stoppage of all insulin delivery)
-  - `*[-]` status events (i.e., suspend & resume)
-    - `[?]` suspensions of insulin delivery are represented as (interval) events with a duration OR
-    - `[?]` suspensions of insulin delivery are represented as pairs of point-in-time events: a suspension and a resumption
-    - `[?]` reason/agent of suspension (`automatic` or `manual`)
-    - `[?]` reason/agent of resumption (`automatic` or `manual`)
+    - `[x]` prime volume in units of insulin
+  - `[x]` reservoir change (or reservoir rewind)
+    - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+  - `[x]` status events (i.e., suspend & resume)
+    - `[ ]` suspensions of insulin delivery are represented as (interval) events with a duration OR
+    - `[x]` suspensions of insulin delivery are represented as pairs of point-in-time events: a suspension and a resumption
+    - `[x]` reason/agent of suspension (`automatic` or `manual`)
+    - `[ ]` reason/agent of resumption (`automatic` or `manual`)
   - calibrations: see [the CGM checklist](CGMChecklist.md) instead
-  - `*[-]` time changes (presence of which is also in the [BtUTC section](#bootstrapping-to-utc) below)
-    - `*[-]` device display time `from` (before change) and `to` (result of change)
+  - `[x]` time changes (presence of which is also in the [BtUTC section](#bootstrapping-to-utc) below)
+    - `[x]` device display time `from` (before change) and `to` (result of change)
     - `[x]` agent of change (`automatic` or `manual`)
     - `[ ]` timezone
     - `[ ]` reason for change (read from device)
 
-Device-specific? (Add any device-specific notes/additions here.)
+##### Device-specific? (Add any device-specific notes/additions here.)
+
+- At the moment we only process the most recent 90 days of log history, as it takes too long to get all the records from pumps with a large history. This means we also need to search for the newest relevant event to start reading from. Events like USB-connected are considered to be not relevant (since we don't process them), but we do consider events like reservoir-change events to be relevant (as they are rendered in the Basics view).
+- An occlusion alarm does not trigger a basal rate change event (used to create all other basals), so a suspended basal is created manually.
+- For status events reasons and causes are provided from the alarms if available, i.e., for occlusion, auto-off, no-insulin and no-power alarms. For cartridge-change alarms, the cause will show as other.
 
 #### SMBG
 
   - `[x]` blood glucose value
   - `[x]` subType (`linked` or `manual`)
-  - `*[?]` units of value (read from device, not hard-coded)
+  - `[ ]` units of value (read from device, not hard-coded)
   - `*[?]` out-of-range values (LO or HI)
   - `*[?]` out-of-range value thresholds (e.g., often 20 for low and 600 for high on BGMs)
```

Comment:
Since you're entering the value manually, I suppose it doesn't make sense to have LO/HI or out-of-range? Out-of-range/threshold info does exist in the spec for CGM data - I've put in a note.